contract/models/abstract_contract.py

```diff
@@ -22,7 +22,12 @@ class ContractAbstractContract(models.AbstractModel):
     partner_id = fields.Many2one(
         comodel_name="res.partner", string="Partner", index=True
     )
-    pricelist_id = fields.Many2one(comodel_name="product.pricelist", string="Pricelist")
+    pricelist_id = fields.Many2one(
+        comodel_name="product.pricelist",
+        string="Pricelist",
+        check_company=True,
```

Comment:
this leads to an issue with the pricelists in contract, when no multi companay is used.
I can no longer choose existing lists.
Before this change it was working fine.